cbdr/cortex
# COR-693, COR-684: Dashboard (Quick Links, News) and Design Refresh

**Purpose:**
Implements the Beta Dashboard w/ Quick Links + News, and refreshes Cortex's design with an approximation of official mockups. Specifically:
* Adds padding to main content
* Redesigns Wizard steps, removing sidebar and step descriptions
* Removes heinous card and nav/header shadows
* Changes background to white across app
* Changes 'Custom Content' wording to 'Beta'
* ...and more!

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/COR-693
https://cb-content-enablement.atlassian.net/browse/COR-684

**Changes:**
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* After
![image](https://cloud.githubusercontent.com/assets/1045168/24091819/7e302244-0d18-11e7-9eed-d6465388f5dd.png)
![image](https://cloud.githubusercontent.com/assets/1045168/24091701/add436c6-0d17-11e7-9cbb-b20c916d07d7.png)

* Before
![image](https://cloud.githubusercontent.com/assets/1045168/24091718/c787e126-0d17-11e7-988e-12c2982544bc.png)

**QA Links:**
http://web.cortex-1.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * http://web.cortex-1.development.c66.me/
    * Dashboard
    * Blog Creation, Media Creation

* Steps to take
  * Login
  * Test Dashboard
  * Test Media Wizard
  * Test Employer Blog Wizard

* Responsive considerations
  * Everything should be responsive except for the `mdl-data-table` that powers the Notes feature - this MDL component is not responsive.

**Relevant PRs/Dependencies:**
N/A

**Additional Information**
Part of 3/17 10% time
